python/lsst/cp/pipe/ptc.py

```diff
@@ -101,14 +101,16 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
         default=1,
     )
     minMeanSignal = pexConfig.Field(
```

Comment:
Can this use `pexConfig.DictField` instead?  This won't have a clean command line interface (I think you need to use a config file to set `DictField` entries), but it would put the type checking into the config object instead of forcing some later parsing.  The lack of a command line interface will also be mitigated by the switch to gen3.
An example of the config field: https://github.com/lsst/pipe_tasks/blob/master/python/lsst/pipe/tasks/ingest.py#L58
An example of setting the values: https://github.com/lsst/obs_lsst/blob/master/config/ucd/ingestCalibs.py#L14

Reply:
Thanks, I'll try this.